app/Models/CustomTour.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class CustomTour extends Model
+{
+    use HasFactory;
+
+    protected $connection = 'tours_db';
+
+    protected $table = 'tours';
```

Comment:
Give the table and class the same name, so you don't have to override the default here.